nemos/dev

```diff
@@ -0,0 +1,274 @@
+ * feature: no-follow-recommends
+
+This seed contains all ECU packages which are considered as debug tools.
+These packages are not qualified for mass production, but only security maintained.
+
+== ECU expected future prod ==
+
+This list of packages is intended to become prod packages in the future.
+
+ * base-files       # Debian base system miscellaneous files
+ * base-passwd      # Debian base system master password and group files
+ * busybox-static	# Standalone rescue shell with tons of builtin utilities
+ * ca-certificates  # Common CA certificates
+ * cryptsetup       # disk encryption support - startup scripts
+ * ebtables         # Ethernet bridge frame table administration
+ * ethtool          # display or change Ethernet device settings
+ * gzip				# GNU compression utilities
+ * iproute2         # networking and traffic control tools
+ * locales          # GNU C Library: National Language (locale) data support
+ * lsb-base         # Linux Standard Base init script functionality
+ * lsb-release      # Linux Standard Base version reporting utility
+ * mount            # tools for mounting and manipulating filesystems
+ * netbase          # Basic TCP/IP networking system
+ * nftables         # Program to control packet filtering rules by Netfilter project
+ * tzdata           # time zone and daylight-saving time data
+ * util-linux       # miscellaneous system utilities
+ * xfsprogs         # Utilities for managing the XFS filesystem
+ * xz-utils			# XZ-format compression utilities
+ * zstd				# fast lossless compression algorithm -- CLI tool
+
+ * systemd          # system and service manager
+ * systemd-sysv     # system and service manager - SysV links
+ * dbus			    # simple interprocess messaging system (daemon and utilities)
+ * udev					# /dev/ and hotplug management daemon
+
+ * crun         # lightweight OCI runtime for running containers
+ * podman       # engine to run OCI-based containers in Pods
+ * libesmtp6
+ * liblog4c3
+ * systemd-coredump
+
+== ECU debug tools ==
```

Comment:
How was it decided what is a debug tool and what is future prod? I would expect some from here like dmsetup or acl also candidates for prod.

Reply:
The prod tools are defined by architecture requirements, i.e. all tools required/used for architecture use cases will be added to prod. For a future release, acl and dmsetup may become a prod tool.

Reply:
Ok, just expected them in future prod.